README.md

```diff
@@ -45,6 +45,52 @@ git clone --recursive https://github.com/Apra-Labs/ApraPipes.git
   * Look at the unit_tests/params_test to check for sample usage of parameters in test code
 
 
+## Prerequisites for macOS
+- Update to Xcode 13.2.1 or Recent
+    - Please note you need atleast 40GB of free space on your macPC to update to latest version
+    - Follow this Link to [Install XCode](https://www.freecodecamp.org/news/how-to-download-and-install-xcode/)
+- Install Homebrew
+    ```sh 
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+    ```
+- Install nproc and coreutils
+    ```sh 
+    brew install coreutils
```

Comment:
comment says nproc and coreutils but the command line below shows only coreutils ?